hooks/addTodayWidgetToProject.js

```diff
@@ -298,8 +367,7 @@ module.exports = function (context) {
               }
               if (projectContainsSwiftFiles) {
                 buildSettingsObj['SWIFT_VERSION'] = '3.0';
-                buildSettingsObj['ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES'] =
-                  'YES';
+                buildSettingsObj['ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES'] = ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES || 'YES';
```

Comment:
We should add this to the table in the readme, default: 'YES', and in the description: changeable to 'NO' (including the apostrophes)